l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -65,7 +66,7 @@ contract MailboxFacet is ZKChainBase, IMailbox {
     /// @inheritdoc IMailbox
     function bridgehubRequestL2Transaction(
         BridgehubL2TransactionRequest calldata _request
-    ) external onlyBridgehub returns (bytes32 canonicalTxHash) {
+    ) external onlyInteropCenter returns (bytes32 canonicalTxHash) {
```

Comment:
wont work during the upgrade (during the time when chains themselves are upgraded, but not the rest of the system)